docs/security.rst

```diff
@@ -60,3 +60,75 @@ Girder returns an error when a Cross-Origin request is made (one with the
 ``Origin`` header) that does not match the system configuration settings.
 Although most modern web browsers also enforce this, some additional security
 is added by enforcing it at the request level.
+
+
+Database Injection Attacks
+--------------------------
+
+Girder defends against database injection attacks by using PyMongo as the only
+pathway between the application server and the database server. This protect
+against many injection vulnerabilities as described in the
+`MongoDB Documentation
+<http://docs.mongodb.org/manual/faq/developers/#how-does-mongodb-address-sql-or-query-injection>`_.
+Girder also uses a model layer to mediate and validate all interaction with
+the database. This ensures that for all database operations and structural
+attributes (collection name, operation type) are hardcoded and not modifiable
+by the client, while data attributes (stored content) are validated for proper
+form before being accepted from a client.
+
+Additionally, we strongly recommend configuring your MongoDB server with
```

Comment:
Change to: "...strongly recommend configuring your MongoDB server with JavaScript..." to make clear that this is not an option within Girder, but rather must be done by the server admin.